### PR TITLE
chore: [FFM-1907]: README Updates for M1 Macs

### PR DESCRIPTION
**Pull request by bmjen**
##### Summary:

Updates to README to support install on M1 based Macbooks

##### Jira Links:

[FFM-1907](https://harness.atlassian.net/browse/FFM-1907)

##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by bmjen** [comment]
retrigger titlecheck

**Review by laura-mannus — APPROVED** [review]
Nice :)

**Review by chrisblakely01 — DISMISSED** [review]

File: docs/M1_macbooks.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+## Additional Installation Steps for M1 based Macbooks
+
+When following the ReadMe there are additional steps required for users with a M1 MacBook. Please see below:
```
Comment (knagurski) [inline docs/M1_macbooks.md:3]:
```suggestion
Additional steps required prior to installation for users with a M1 Mac. Please see below:
```

**Review by knagurski — COMMENTED** [review]
Some suggestions 👍🏻

File: docs/M1_mac.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+## Additional Installation Steps for M1-based Macs
+
+When following the README there are additional steps required for users with a M1 MacBook. Please see below:
```
Comment (knagurski) [inline docs/M1_mac.md:3]:
```suggestion
When following the README there are additional steps required for users with a M1 Mac. Please see below:
```

**Review by knagurski — DISMISSED** [review]

**Review by laura-mannus — APPROVED** [review]

**Pull request merged**